vlo-web-app/src/main/java/eu/clarin/cmdi/vlo/wicket/pages/FacetedSearchPage.java

```diff
@@ -154,6 +161,7 @@ public void setObject(FacetSelectionType object) {
                 super.setObject(object);
                 //persist in session
                 VloWebSession.get().setFacetSelectionTypeMode(object);
+                String ip = ((WebClientInfo)VloWebSession.get().getClientInfo()).getProperties().getRemoteAddress();
```

Comment:
does this do anything?

Reply:
Not it doesn't, it was part of deleted code.
Corrected!